components/blitz/src/ome/services/blitz/repo/ManagedImportRequestI.java

```diff
@@ -661,7 +661,10 @@ public MessageDigest parseData(
                 store.setChannelGlobalMinMax(
                         c, minMax[0], minMax[1], series);
             }
-            return null;
+            if (pixelType != FormatTools.FLOAT &&
```

Comment:
from what I gather (not my code), the global min and max are set to the min/max of the pixel type.
For large planes, the planes are not read (i.e. return null).In the case of float/double we do not generate pyramid so do not read any planes so the values are always set to min/max of pixel range.
The check added is to keep the same behaviour for other pixels types